src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -67,7 +111,7 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
     const max = this.props.numSlices[activeAxis];
     const currentLeftSliderValue = Math.round(this.props.region[activeAxis][0] * max);
     const leftValue = (currentLeftSliderValue + delta + max) % max;
-    this.updateClipping(activeAxis, leftValue, leftValue);
+    this.updateClipping(activeAxis, leftValue, leftValue + 1);
```

Comment:
Confused about why this changed.  Is this resolving a bug in single slice mode?

Reply:
Yes, this seems to be something I missed when changing this component during the functional refactor. Used to be that the min and max were offset by 1 automatically. Now they're not, so moving the slice with the step/play buttons would flatten it down to nothing. (Ought to take a look now or between milestones at whether slider numbers still make as much sense as they ought to.)